test/recipient_test.rb

```diff
@@ -0,0 +1,105 @@
+require_relative 'test_helper'
+
+describe "Recipient" do
+  describe 'constructor' do
+    it "creates a Recipient object" do
+      test = Recipient.new(slack_id: "C01BKP7MWNB",name: "random")
+      expect(test).must_be_kind_of Recipient
+      expect(test.slack_id).must_equal "C01BKP7MWNB"
+      expect(test.name).must_equal "random"
+    end
+  end
+
+  describe 'send_message' do
+    before do
+      # slackbot will be in every slack workspace
+      @test = Recipient.new(slack_id: "U01BKP7MGVD",name: "pauline.chane")
+    end
+    it 'returns message hash for successful messages for users' do
+      VCR.use_cassette("send message to recipient") do
+        message = "maracuyA"
+        response = @test.send_message(message)
+        expect(response).must_be_instance_of Hash
+        expect(response['message']['text']).must_equal message
+        expect(response['channel']).must_equal "D01CD4P11J5"
+      end
+    end
+    it 'returns message hash for successful messages for channels' do
+      VCR.use_cassette("send message to recipient") do
+        channel = Recipient.new(slack_id: "C01BKP7MWNB",name: "random")
+        message = "maracuyA"
+        response = channel.send_message(message)
+        expect(response).must_be_instance_of Hash
+        # for messages to channels
+        expect(response['channel']).must_include channel.slack_id
+      end
+    end
+    it "rejects messages longer than 4000 characters" do
+      VCR.use_cassette("send message to recipient") do
+        message = ""
+        4005.times do
+          message += "A"
+        end
```

Comment:
Ruby actually has built in syntax for this (for some reason):

```suggestion
        message = "A" * 4005
```